spec/spec_helper_acceptance.rb

```diff
@@ -1,38 +1,20 @@
-ENV['PUPPET_INSTALL_TYPE'] ||= 'agent'
-ENV['BEAKER_IS_PE'] ||= 'no'
-ENV['BEAKER_PUPPET_COLLECTION'] ||= 'puppet6'
-ENV['BEAKER_debug'] ||= 'true'
+require 'voxpupuli/acceptance/spec_helper_acceptance'
+
 ENV['BEAKER_setfile'] ||= 'centos7-64{hostname=centos7-64.example.com}'
-ENV['BEAKER_HYPERVISOR'] ||= 'docker'
-
-require 'beaker-puppet'
-require 'beaker-rspec'
-require 'beaker/puppet_install_helper'
-require 'beaker/module_install_helper'
-
-run_puppet_install_helper unless ENV['BEAKER_provision'] == 'no'
-install_module_on(hosts)
-install_module_dependencies_on(hosts)
-
-RSpec.configure do |c|
-  # Readable test descriptions
-  c.formatter = :documentation
-
-  # Configure all nodes in nodeset
-  c.before :suite do
-    # Install module and dependencies
-    hosts.each do |host|
-      if fact_on(host, 'osfamily') == 'RedHat'
-        # don't delete downloaded rpm for use with BEAKER_provision=no +
-        # BEAKER_destroy=no
-        on host, 'sed -i "s/keepcache=.*/keepcache=1/" /etc/yum.conf'
-        # refresh check if cache needs refresh on next yum command
-        on host, 'yum clean expire-cache'
-        # We need SCL as well for dynflow
-        if fact_on(host, 'operatingsystem') == 'CentOS'
-          host.install_package('centos-release-scl-rh')
```

Comment:
We need this still

Reply:
Yes, haven't gotten around to it. Working on a better pattern to combine modulesync and local module overrides.